src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,16 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
```

Comment:
I think adding a header here would be better 🤔

Reply:
Yep, will add more headers later. Thank you for the feedback! :)